### DISCUSSION TITLE
It is very annoying to use ::before and ::after

### DISCUSSION THREAD

**Discussion by reza869** [tailwindlabs/tailwindcss#8215] [Ideas]
Using ::before and ::after in front of every line of CSS is very annoying. Ex.:
`<span class="before:absolute before:top-1/2 before:left-1/2 before:w-[6px] before:h-[6px] before:conent-[''] before:transition-all before:duration-500 before:-translate-x-1/2 before:-translate-y-1/2 before:transform before:bg-[#cccccc] before:rounded-full"></span>`

I hope you will make it easier in the next update. Thanks

**Comment by jenstornell** [comment]
I agree, it's a mess because of all the classes. Personally I use @apply for these things. I'm not sure I could be solved in any other way.
